submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/db/reactive/repositories/Repositories.kt

```diff
@@ -67,8 +67,13 @@ interface SubmissionMongoRepository : CoroutineCrudRepository<DocSubmission, Obj
     suspend fun findSubmissionCollections(accNo: String): SubmissionCollections?
 }
 
-suspend fun SubmissionMongoRepository.getByAccNo(accNo: String): DocSubmission =
-    findByAccNo(accNo) ?: throw SubmissionNotFoundException(accNo)
+suspend fun SubmissionMongoRepository.getByAccNo(accNo: String): DocSubmission {
+    return findByAccNo(accNo) ?: throw SubmissionNotFoundException(accNo)
```

Comment:
I will include version SubmissionNotFoundException so we can latter identify issue